src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizard.cmp

```diff
@@ -180,77 +182,82 @@
-                                        <lightning:input label="{!field.label + ': ' + $Label.c.lblRequired}"
-                                                         type="checkbox"
-                                                         checked="{!field.requiredInEntryForm}"
-                                                         disabled="{!field.alwaysRequired}"
-                                                         aria-disabled="{!field.alwaysRequired}"
-                                                         variant="label-hidden"/>
+                                    <lightning:layoutItem size="2" class="slds-text-align_center slds-p-horizontal_x-small slds-form-element__legend">
+                                        {!$Label.c.lblRequired}
                                     </lightning:layoutItem>
-
                                 </lightning:layout>
 
+                                <aura:iteration items="{!currentFieldGroup.fields}" var="field">
+
+                                    <lightning:layout horizontalAlign="center" multipleRows="true" verticalAlign="center">
+                                        <!-- Field Label-->
+                                        <lightning:layoutItem size="4" class="slds-text-align_left slds-p-around_x-small">
+                                            <span aria-hidden="true">
+                                                {!field.label}
+                                            </span>
+                                        </lightning:layoutItem>
+
+                                        <!-- Field Default Value -->
+                                        <lightning:layoutItem size="4" class="slds-text-align_left slds-p-around_x-small">
+
+                                            <aura:if isTrue="{!greaterthan(field.options.length, 0)}">
+                                                <lightning:select aura:id="defaultValueField"
+                                                                  label="{!$Label.c.stgDefaultValue}"
+                                                                  value="{!field.defaultValue}"
+                                                                  variant="label-hidden"
+                                                                  class="hideLabel">
+                                                    <option value=""></option>
+                                                    <aura:iteration items="{!field.options}" var="opt">
+                                                        <option value="{!opt.value}">{!opt.label}</option>
+                                                    </aura:iteration>
+                                                </lightning:select>
+                                                <aura:set attribute="else">
+                                                    <!-- TODO: ensure these are accessible -->
```

Comment:
Is there additional testing required here?

Reply:
Update: testing is done, and we do need to add an aria label here because the field label is being lost in this format.